### PR TITLE
Bring configuration up to date

### PR DESCRIPTION
**Pull request by swissspidy**
* Update dependencies
* Require PHP 7.3
* Add GitHub Actions workflow

### CODE REVIEW

**Comment by swissspidy** [comment]
```
Fatal error: Uncaught Error: Class 'Yoast\PHPUnitPolyfills\TestCases\TestCase' not found in /home/travis/build/php-stubs/wordpress-tests-stubs/wordpress-tests-stubs.php:22
```

This is because core uses https://github.com/Yoast/PHPUnit-Polyfills/ and thus any plugin using the WordPress test suite also needs to use it.

**Comment by szepeviktor** [comment]
@swissspidy Would you like to be the maintainer?

(expected answer: Yes)

**Comment by swissspidy** [comment]
Sure, why not 😅 

While you're here, any idea why the abstract `WP_UnitTestCase_Base` class might be missing from the generated stubs?

**Comment by szepeviktor** [comment]
> any idea why

That is the reason you'll be a maintainer!! :)

**Comment by szepeviktor** [comment]
> While you're here, any idea why the abstract `WP_UnitTestCase_Base` class might be missing from the generated stubs?

Is it there in v5.8.1?

**Comment by swissspidy** [comment]
Yes it is:

https://github.com/php-stubs/wordpress-tests-stubs/blob/0aef1eda211ab982bb693bde25eb27dfeb1e8fa6/wordpress-tests-stubs.php#L18-L32

**Comment by szepeviktor** [comment]
Okay.
Then it has disappeared in one of the above commits.

**Comment by szepeviktor** [comment]
GitHub shows much of the stubs file has been deleted in this PR.
kép

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "homepage": "https://github.com/php-stubs/wordpress-tests-stubs",
     "license": "MIT",
     "require-dev": {
-        "php": "~7.1",
+        "php": "~7.1 || ~8.0",
         "giacocorsiglia/stubs-generator": "^0.5.0"
```
Comment (szepeviktor) [inline composer.json]:
```suggestion
        "php": "~7.1 || ~8.0",
        "php-stubs/generator": "^0.8.0"
```

Comment (szepeviktor) [reply composer.json]: This repo needs so much modernization.

**Comment by szepeviktor** [comment]
We have this mysterious https://github.com/php-stubs/wordpress-tests-stubs/blob/master/extended-classes.php and the upgrade to Composer v2.

**Pull request merged**